platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -257,106 +227,101 @@ public void testConstructorWithNullConfigurationFileExtension() {
                 null);
     }
 
-    @Test
-    public void testInitDoesNotCreateExistingProcessedAndFailedAndExportedDirectories()
-            throws IOException {
-        when(processedDirectory.exists()).thenReturn(true);
-        when(processedDirectory.isDirectory()).thenReturn(true);
-        when(failedDirectory.exists()).thenReturn(true);
-        when(failedDirectory.isDirectory()).thenReturn(true);
+    @Test(expected = IOException.class)
+    public void testInitDoesNotCreateExistingProcessedAndFailedDirectories() throws IOException {
 
-        ConfigurationAdminMigration configurationAdminMigration =
-                createConfigurationAdminMigratorWithNoFiles();
+        PowerMockito.doNothing()
```

Comment:
Can you try and see what happens when you remove all those `PowerMockito.doNothing()` and `FileUtils.forceMkDir()` expectations. You may only need the one expectation that throws the exception here.